### ISSUE TITLE
Check that output shape must be [n_classes x h x w]

### ISSUE THREAD

**Issue by ADPriceless** [ADPriceless/TDA_FinalProject_Solar#15]
Expansion of last task in Issue #3: "how to format targets to be used by pre-trained model":
- [x]  Check documentation
- [x] If that fails, create a test

**Comment by ADPriceless** [comment]
Documentation for CrossEntropyLoss states that the shape of the target should be n classes for dim=1
